src/tests/nativeaot/SmokeTests/HardwareIntrinsics/Program.cs

```diff
@@ -50,100 +50,208 @@ static int Main()
         //
         // The test is compiled with multiple defines to test this.
```

Comment:
@MichalStrehovsky Could you please review the change in the native AOT smoke tests?